plugins/mcreator-core/procedures/world_data_isday.json

```diff
@@ -1,12 +1,4 @@
 {
-  "args0": [
```

Comment:
This procedure sometimes works strangely on the client-side (not returning true when it should). Keep this, there were quite some bug reports in the past due to this indicator not being present.